### ISSUE TITLE
virtualenv setup failure during install test on Python 3.7 with minimum package levels

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#2174]
The Travis run on OS-X on Python 3.7.7 with minimum package levels fails when setting up the virtualenv for an install test (see https://travis-ci.org/github/pywbem/pywbem/jobs/674559955):

```
Testcase test1: Pip install from repo root directory: tests/installtest/../..
Saving current virtualenv: /Users/travis/venv
Creating virtualenv: tests/installtest/../../tmp_installtest/virtualenvs/pywbem_test_test1
..using this Python command: /Users/travis/venv/bin/python

Error: Command failed with rc=100: virtualenv -p /Users/travis/venv/bin/python tests/installtest/../../tmp_installtest/virtualenvs/pywbem_test_test1, output follows:

/Users/travis/venv/lib/python3.7/site-packages/virtualenv.py:991: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
Using base prefix '/usr/local/opt/python/bin/../Frameworks/Python.framework/Versions/3.7'
New python executable in /Users/travis/build/pywbem/pywbem/tmp_installtest/virtualenvs/pywbem_test_test1/bin/python
ERROR: The executable /Users/travis/build/pywbem/pywbem/tmp_installtest/virtualenvs/pywbem_test_test1/bin/python is not functioning
ERROR: It thinks sys.prefix is '/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7' (should be '/Users/travis/build/pywbem/pywbem/tmp_installtest/virtualenvs/pywbem_test_test1')
ERROR: virtualenv is not compatible with this system or executable
Running virtualenv with interpreter /Users/travis/venv/bin/python
make: *** [installtest] Error 1
```

Since this was a run with minimum package levels, it used virtualenv 14.0.0. That version does not state support for Python 3.7 yet.

**Comment by andy-maier** [comment]
The Python versions supported by virtualenv as stated on PyPI via the "Programming Language" classifiers, are:
* virtualenv 14.0.0: py26-27, py31-34
* virtualenv 15.0.0: py26-27, py33-35
* virtualenv 16.0.0: py27, py34-36
* virtualenv 16.1.0: py27, py34-37 (up to 16.7.10 which is latest before 20.0.0)
* virtualenv 20.0.0 (rewrite): py27, py34-38

**Comment by andy-maier** [comment]
This will be fixed by increasing the virtualenv minimum version to the first version that states support for the Python version in question.

**Comment by andy-maier** [comment]
Fixed in PR #2175 for master, and TBD for 0.17.2.
